### PR TITLE
Make retrieved at required

### PR DESCRIPTION
**Pull request by bensymonds**

### CODE REVIEW

**Comment by bensymonds** [comment]
@skenaja can you approve the schema change please?
@pikesley can you approve the Travis changes please (feel free to comment on the schema change too, but I think @skenaja approval is enough so you're not required).

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,13 @@ cache: bundler
...
   - 2.0.0
-  - 2.1.0
-  - 2.2.0
-  - 2.3.0
+  - 2.1
+  - 2.2
+  - 2.3
+  - 2.4
+  - 2.5
+  - 2.6
```
Comment (pikesley) [inline .travis.yml]: Are we being a bit keen here? 2.6 is still in preview

Comment (bensymonds) [reply .travis.yml]: Meh I figured we may as well test against it anyway, to get advance warning of any issues. It's the point of being "in preview" I guess..?

Comment (pikesley) [reply .travis.yml]: And I guess it's passing in Travis?

Comment (bensymonds) [reply .travis.yml]: So far...

**Comment by skenaja** [comment]
OK with schema changes. Looks like openc_bot sets retrieved at [by default](https://github.com/openc/openc_bot/blob/911a9fed01f183188bb7a46fd4cdee2a075ac851/lib/openc_bot/helpers/register_methods.rb#L233) if it's not provided in an external bot.

**Comment by bensymonds** [comment]
> Looks like openc_bot sets retrieved at [by default](https://github.com/openc/openc_bot/blob/911a9fed01f183188bb7a46fd4cdee2a075ac851/lib/openc_bot/helpers/register_methods.rb#L233) if it's not provided in an external bot.

Hmm yeah. I'm not sure about that. Going to hold off merging this until we've had a think about it.

**Comment by bensymonds** [comment]
I haven't really thought about the openc_bot thing yet... apart from coming to the conclusion that it's independent of this schema change (irrespective of whether openc_bot is setting `retrieved_at`, we still want to make it required in the schema). So I will merge this shortly. I made a [separate ticket](https://opencorporates.atlassian.net/browse/OCD-1071) to track the openc_bot question.

**Pull request merged**